### ISSUE TITLE
pandas.Timestamp as an annotation causes TypeError

### ISSUE THREAD

**Issue by xbanke** [seandstewart/typical#70]
* typical version: 2.0.8
* Python version: 3.7.4
* Operating System:

### Description
`pandas.Timestamp` is a sub-class of `datetime`, but as an annotation will cause TypeError:
```py
import typic
from datetime import datetime
import pandas as pd


issubclass(pd.Timestamp, datetime)  # True


@typic.klass
class Foo:
    dt: pd.Timestamp
# TypeError: __init__() got an unexpected keyword argument 'additionalProperties'
```

This is OK at version `2.0.5`. 
Personally, I think `pandas.Timestamp` is more convenient than builtin `datetime` in data analysis.

**Comment by seandstewart** [comment]
This is a regression due to expanding awareness of subtypes. Should have a fix shortly.

**Comment by seandstewart** [comment]
This has been resolved in v2.0.9

**Comment by xbanke** [comment]
Thanks.
